cli/js/web/console_table.ts

```diff
@@ -4,8 +4,6 @@
 import { TextEncoder } from "./text_encoding.ts";
```

Comment:
```suggestion

```

Since the line ```const encoder = new TextEncoder(); ``` is no longer used and nowhere else found in the code, then the import could be removed aswell, unless it casts some kind of side effects, which I don't know if it does, but I believe it does not.

Reply:
thanks, you are right